shiny/_main.py

```diff
@@ -483,6 +483,7 @@ def try_import_module(module: str) -> Optional[types.ModuleType]:
     "Dashboard": "dashboard",
     "Multi-page app with modules": "multi-page",
     "Custom JavaScript Component": "js-component",
+    "Choose from online gallery 🔗": "external-gallery",
```

Comment:
Maybe this could be "Choose from Shiny Templates gallery online"? It's hard to tell from "online gallery" that it's **our** online gallery. As written it kind of feels like there would be another choice to pick which online gallery.

While here I also noticed that we don't indicate which options will lead to more choices to pick the template. We should add `...` where there's a submenu to pick the template, e.g. `Custom JavaScript component...`.

Oh and we should use consistent casing and emoji on these: they should all be sentence case and if we start using emojis we should use them for all options.

Reply:
Thanks, good points. 

I'm trying to keep this PR aimed at just the addition of this choice, and https://github.com/posit-dev/py-shiny/pull/1274 already addresses the casing issue, so I'll address the submenu thing there as well

Reply:
I'm ok with the link emoji as it hints it's going to do something different